provider/kubernetes/kubernetes.go

```diff
@@ -308,6 +310,50 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				}
 			}
 		}
+
+		for _, t := range i.Spec.TLS {
+			tlsSecret, exists, err := k8sClient.GetSecret(i.Namespace, t.SecretName)
+			if err != nil {
+				log.Errorf("Unable to retrive secret %s/%s: %s", i.Namespace, t.SecretName, err)
+				continue
+			}
+			if !exists {
+				log.Warnf("Unable to find secret %s/%s", i.Namespace, t.SecretName)
+				continue
+			}
+			_, tlsCrtExists := tlsSecret.Data["tls.crt"]
+			_, tlsKeyExists := tlsSecret.Data["tls.key"]
```

Comment:
We seem to need both ignored variables later, so let's store them here for reusal below.